tests/appsec/waf/test_telemetry.py

```diff
@@ -209,6 +209,9 @@ def _validate_headers(headers, request_type):
         "DD-Client-Library-Language": expected_language,
         "DD-Client-Library-Version": "",
     }
+    if expected_language == "python":
```

Comment:
on line 207, maybe this will be more elegant : 

```python
"DD-Telemetry-API-Version": "v2" if expected_language == "python" else "v1",  # small explanation why python uses v2
```